src/api/index.js

```diff
@@ -95,6 +95,13 @@ export const updateComment = ({
   const timestampId = `${new Date().getTime()}`;
   const userComment = userCommentText && userCommentText.trim();
 
+  if (userComment === '' || userComment === null) {
```

Comment:
Is there a reason that you're specifically checking for this instead of just doing `!userComment`? You don't want to check for false?